addons/packages/ako-operator/1.5.0/README.md

```diff
@@ -17,6 +17,16 @@ in the future.
 |:---:|:---:|:---:|:---:|
 |  ❌ |   ❌ | ✅  |  ❌  |
 
+## Mailing lists
+
+* Use [tkg-infrax-akita](mailto:tkg-infrax-akita@groups.vmware.com) to report security concerns to the AKO Team,
```

Comment:
Same here for the mailing list, we can let them send mail to this address, but we can let them join in an internal mailing group.